src/index.js

```diff
@@ -0,0 +1,61 @@
+function randomNumber() {
+  return Math.floor(Math.random() * 10);
+}
+
+function isInvalid(carNames) {
+  const names = carNames.split(',').map((name) => name.trim());
+  return names.some((name) => name === '' || name.length > 5);
+}
+
+function carPositions(cars) {
+  cars.forEach((car) => {
+    console.log(`${car.name}: ${'-'.repeat(car.position)}`);
+  });
+}
+
+function getRoundResult(cars) {
+  const roundResults = [];
+  cars.forEach((car) => {
+    const randomValue = randomNumber();
+    if (randomValue >= 4) {
+      car.position++;
+    }
+    roundResults.push(`${car.name}: ${'-'.repeat(car.position)}`);
+  });
+  return roundResults;
```

Comment:
roundResults 라는 변수를 외부 스코프로 두어서 부수 효과를 발생시키기보다, map 구문으로 배열을 반환하는 것이 더 좋을 것 같은데 어떠실까요?

```suggestion
  const roundResults = cars.map((car) => {
    const randomValue = randomNumber();
    if (randomValue >= 4) {
      car.position++;
    }
    return `${car.name}: ${'-'.repeat(car.position)}`;
  });
```